### ISSUE TITLE
Show spatial extent of manuscripts on a map

### ISSUE THREAD

**Issue by nuest** [GeoinformationSystems/optimap#43]
**[<img src="https://avatars.githubusercontent.com/u/1325054?s=17&v=4" width="17" height="17"> @nuest](https://github.com/nuest)** cloned issue [ifgi/optimetaPortal#13](https://github.com/ifgi/optimetaPortal/issues/13) on 2022-04-18: 

- ~~https://kitcharoenp.github.io/gis/2018/06/26/geodjango_with_openlayers.html~~
- ~~OpenLayers~~ we go with Leaflet!
- https://www.paulox.net/2021/07/19/maps-with-django-part-2-geodjango-postgis-and-leaflet/
- http://blog.mathieu-leplatre.info/geodjango-maps-with-leaflet.html
- [x] the test geometry from is shown on a full page map
- [x] clicking on the geometry opens a popup with the article title, and the title links to an external page

**Comment by nuest** [comment]
This is already working but should be double checked.

**Comment by nuest** [comment]
@BharatVe and @uxairibrar are both running the application locally now - can you please confirm that the test geometries are shown and that you can open a popup by taking a screenshot? Thanks!

**Comment by uxairibrar** [comment]
Yes, it is showing a pop up, when we click on any test geometry, a pop-up appears which displays the article title, time period, and an external link. 

<img width="464" alt="Image" src="https://github.com/user-attachments/assets/fe0c77f7-2ff2-4297-8e90-9d6b05db464b" />

Is it the correct form of it? Or we need further changes in it? @nuest

**Comment by nuest** [comment]
This is fine for now - we can improve the popup in future sprints.
